server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/application-old.scss

```diff
@@ -1444,3 +1444,17 @@ a:hover#link-to-this-page {
     float: left;
 }
 
+#raw-console-file a:link,
+#raw-console-file a:visited {
+  padding-left:16px;

```

Comment:
Properties should be ordered float, font-size, padding-left, padding-right<br>Colon after property should be followed by one space